src/app/helptext/sharing/iscsi/iscsi.ts

```diff
@@ -81,23 +81,30 @@ export const helptext_sharing_iscsi = {
 
   portal_form_placeholder_delete: T("Delete"),
 
-  initiator_form_placeholder_initiators: T("Initiators"),
+  initiator_form_tooltip_connected_initiators: T(
+	'Initiators currently connected to the system. Shown\
+ in IQN format and IP address. Set initiators and click an\
+ <b>-> (arrow)</b> to add the initiators to either the\
+ <i>Allowed Initiators</i> or <i>Authorized Networks</i>\
+ lists. Clicking <b>Refresh</b> updates the\
+ <i>Connected Initiators</i> list.'
+  ),
+
+  initiator_form_placeholder_initiators: T("Allowed Initiators (IQN)"),
   initiator_form_tooltip_initiators: T(
-    "Use <i>ALL</i> keyword or a list of initiator hostnames separated by spaces."
+    "Initiator hostnames. Enter an initiator hostname and press the\
+ <b>+</b> button to add a host."
   ),
 
   initiator_form_placeholder_auth_network: T("Authorized Networks"),
   initiator_form_tooltip_auth_network: T(
-    'Network addresses which can use this initiator. Use\
- <i>ALL</i> or list network addresses with a\
- <a href="https://en.wikipedia.org/wiki/Classless_Inter-Domain_Routing"\
- target="_blank">CIDR</a> mask. Separate multiple\
- addresses with a space:\
- <i>192.168.2.0/24 192.168.2.1/12</i>.'
+    'Network addresses which can use this initiator. Enter an IPv4 or\
```

Comment:
What happened to the updated text about the optional CIDR netmask?

Reply:
Got ripped out when I rewrote the messages. I'll add it back.